CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # [Unreleased](https://github.com/pybamm-team/PyBaMM/)
 
+- Deprecated `pybamm.Simulation.set_parameters` and `pybamm.Simulation.set_up_and_parameterise_experiment` functions in `pybamm.simulation.py`. ([#3752](https://github.com/pybamm-team/PyBaMM/pull/3752))
+
```

Comment:
This should probably be in breaking changes since it is intended as a deprecation.